File: 167-two-sum-ii-input-array-is-sorted/167-two-sum-ii-input-array-is-sorted.py
class Solution(object):
    def twoSum(self, numbers, target):
        """
        :type numbers: List[int]
        :type target: int
        :rtype: List[int]
        """
        ans = []
        
        low = 0
        high = len(numbers)-1
        
        while low < high:
            if numbers[low] + numbers[high] == target:
                ans.append(low+1)
                ans.append(high+1)
                return ans
            elif numbers[low] + numbers[high] < target:
                low += 1
            else:
                high -=1
        
        return ans
                

File: 167-two-sum-ii-input-array-is-sorted/167-two-sum-ii-input-array-is-sorted.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        int left,right;
        left = 0;
        right = numbers.length-1;
        int[] ans = new int[2];
        while (left < right){
            if (numbers[left] + numbers[right] == target){
               ans[0] = left+1;
                ans[1] = right + 1;
                
                return ans;
                
            }
            else if((numbers[left] + numbers[right] < target)){
                left ++;
            }
            else{
                right --;
            }
        }
        
        
        return ans;
    }
}